spec/services/push_notifications/send_spec.rb

```diff
@@ -59,31 +60,33 @@
   end
 
   context "with devices for multiple users" do
-    let(:consumer_app) { create(:consumer_app) }
+    let(:consumer_app_ios) { create(:consumer_app, platform: Device::IOS) }
+    let(:consumer_app_android) { create(:consumer_app, platform: Device::ANDROID) }
 
     before do
-      create(:device, user: user, consumer_app: consumer_app)
-      create(:device, user: user2, consumer_app: consumer_app)
+      create(:device, user: user, consumer_app: consumer_app_ios)
+      create(:device, user: user2, consumer_app: consumer_app_ios)
+      create(:device, user: user2, consumer_app: consumer_app_android, platform: Device::ANDROID)
     end
 
     it "creates a notification and enqueues it" do
-      mocked_objects = mock_rpush(consumer_app)
+      mocked_objects = mock_rpush(consumer_app_ios)
 
-      described_class.call(**many_targets_params)
+      described_class.call(**params)
 
-      expect(mocked_objects[:rpush_notification]).to have_received(:save!).exactly(2).times
+      expect(mocked_objects[:rpush_notification]).to have_received(:save!).exactly(1).times
```

Comment:
Just a minor RSpec style thing:

```suggestion
      expect(mocked_objects[:rpush_notification]).to have_received(:save!).once
```

Or, as per the [documentation](https://relishapp.com/rspec/rspec-mocks/docs/setting-constraints/receive-counts):

> If you don't specify an expected receive count, it defaults to once.

I do like the explicit `once` though.